src/app/grid/grid-select/grid-select-sample.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="grid__wrapper">
+    <div class="sample__header">
```

Comment:
This container is not needed & can be removed.

Reply:
Removed.

Reply:
Fix the tabulation pls:

<div class="grid__wrapper">
        <igx-grid #grid1 [data]="data" [primaryKey]="'name'" width="780px" height="600px">